spec/lib/restrict_admin_by_ip_middleware_spec.rb

```diff
@@ -50,6 +50,18 @@
     expect(body).to eq ["Forbidden"]
   end
 
+  it "handles IPv6 addresses" do
```

Comment:
Does this work if we have a IPv6 address in the list of allowed IPs?

Reply:
Yeah I think so, IPv6 is included in the default developer `ALLOWED_IPS`: https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/f7fa8b18f8a3ba911dae89d54de0035ffddea11d/.env.development#L10

Reply:
This fix fixes a fix for the an issue where ports were being passed to `client_ip` IPv4 IP addresses :)

Reply:
Ah, course